README.md

```diff
@@ -5,7 +5,9 @@ Servable is an implementation of a "simple" observable
 ## To install run:
 
 ```
-npm i servable --save
+npm i servable
+// or, if you're awesome
```

Comment:
Consider breaking out of the code here (and use different language as everyone may not agree with you ;-P)